kibana/plan.sh

```diff
@@ -0,0 +1,36 @@
+pkg_name=kibana
+pkg_version=4.5.1
+pkg_origin=core
+pkg_license=('Apache-2.0')
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_source=https://github.com/elastic/${pkg_name}/archive/v${pkg_version}.tar.gz
```

Comment:
Should we always be preferring github urls over a published download URL for a project, like https://download.elastic.co/kibana/kibana/kibana-4.5.1-linux-x64.tar.gz ?